kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStateListRow.js

```diff
@@ -1,157 +1,175 @@
 ], function(
     Promise,
-    html
+    html,
+    Events,
+    $
 ) {
     'use strict';
 
+    var container;
+
     const t = html.tag,
         div = t('div'),
         td = t('td'),
         span = t('span'),
-        cssBaseClass = 'kb-job-state';
+        a = t('a'),
+        i = t('i'),
+        cssBaseClass = 'kb-job-status',
+        events = Events.make();
 
     function createStateCell(jobState) {
-        let label, icon;
+        let label;
         switch (jobState) {
             case 'completed':
-                label = 'success';
-                icon = 'fa fa-check';
+                label = 'Success';
                 break;
             case 'created':
+            case 'estimating':
             case 'queued':
-                label = jobState;
-                icon = 'fa fa-angle-double-right';
+                label = 'Queued';
                 break;
             case 'running':
-                label = jobState;
-                icon = 'fa fa-spinner';
+                label = 'Running';
                 break;
             case 'error':
+                label = 'Failed';
+                break;
             case 'terminated':
-                label = jobState;
-                icon = 'fa fa-times';
+                label = 'Cancelled';
                 break;
             case 'does_not_exist':
-                label = 'does not exist';
-                icon = 'fa fa-question';
+                label = 'Does not exist';
                 break;
-            case 'estimating':
-            default:
-                label = jobState;
-                icon = 'fa fa-question';
         }
 
         return td({
-            class: `${cssBaseClass}__cell--status`,
+            class: `${cssBaseClass}__cell`
         }, [
             span({
-                class: `${icon} ${cssBaseClass}__icon--${jobState}`
+                class: `fa fa-circle ${cssBaseClass}__icon--${jobState}`
             }),
             ' ' + label
         ]);
     }
 
-    function createActionCell(jobState) {
+    function selectRow(e) {
+        let $currentRow = $(e.target).closest('tr');
+
+        let $allRows = $('.kb-job-status__row');
+
+        // unselect previously selected row
+        $allRows.removeClass(`${cssBaseClass}__row_selected`);
+        $allRows.find('.selected_log').css('display', 'none');
+
+        // add clasees to selected row
+        $currentRow.addClass(`${cssBaseClass}__row_selected`);
+        $currentRow.find('.selected_log').css('display', 'inline');
+    }
+
+    function unselectRow(e) {
+        let $currentRow = $(e.target).closest('tr');
+
+        $(e.target).closest('tr').removeClass(`${cssBaseClass}__row_selected`);
+        $currentRow.find('.selected_log').css('display', 'none');
+    }
+
+    function createActionCell(jobState){
         let label;
         switch (jobState) {
             case 'completed':
-                label = 'Go to results';
+                label = 'GO TO RESULTS';
```

Comment:
GO THERE!

Reply:
I'd be tempted to use a css transform for this, rather than hard-coding the all-caps; it's a presentation issue and using css makes it easier if we decide to change over to using sentence case / lower case / small caps / whatever.